src/App/VPSScan/Scan/ScotlandYard.hs

```diff
@@ -1,29 +1,33 @@
 module App.VPSScan.Scan.ScotlandYard
-  ( HTTP (..),
-    runHTTP,
-    ScanResponse (..),
-    createScotlandYardScan,
-    uploadIPRResults,
+  ( createScotlandYardScan
+  , uploadIPRResults
+  , ScanResponse (..)
+  , ScotlandYardOpts (..)
   )
 where
 
 import App.VPSScan.Types
+import App.VPSScan.Scan.Core
 import Control.Effect.Diagnostics
 import Control.Monad.IO.Class (MonadIO)
 import Data.Aeson
-import Data.Text (Text)
```

Comment:
[nit] let's keep this import as-is -- if you need functions from `Data.Text`, e.g., `pack` also add:
```hs
import qualified Data.Text as T
```
and use `T.pack`

there are a bunch of overlapping function names in `Data.Text` -- so it's meant to be imported qualified